src/backend/x11/window.rs

```diff
@@ -210,7 +210,7 @@ impl WindowBuilder {
         let id = conn.generate_id()?;
         let setup = conn.setup();
 
-        let env_dpi = std::env::var("DRUID_X11_DPI")
+        let env_dpi = std::env::var("GLAZIER_X11_DPI")
```

Comment:
Who is the intended user of this?

If it's end users, it shouldn't be locked behind X11 - it should also work for Wayland when fractional scaling isn't supported (i.e. `GLAZIER_FALLBACK_SCALE`)

If just developers, I suppose it's fine to have, but it just seems a bit messy.

Reply:
As I remember this was intended for Druid developers debugging it.

Reply:
This is renamed and a note added about the intended use of this.